Specification/1._Introduction.md

```diff
@@ -29,24 +25,22 @@ The latter imposes severe restrictions on the amount of logic needed to implemen
 for common high-level functions, such as network discovery, node configuration, node firmware update,
 node status monitoring (which naturally grows into a vehicle-wide health monitoring),
 network-wide time synchronization, dynamic node ID allocation (a.k.a. plug-and-play), etc.
-* **Open specification and reference implementation** - the UAVCAN specification is open and freely available
-for anyone; the reference implementations are distributed under the terms of the
+* **Open specification and reference implementation** - the UAVCAN specification is open and freely available; the reference implementations are distributed under the terms of the
 [MIT License](http://en.wikipedia.org/wiki/MIT_License).
 
 ## Specification update and approval process
 
-UAVCAN development team is charged with advancing the specification based on the input from adopters
-gathered via the [mailing list](/Contact), where everyone can contribute to the discussion.
+The UAVCAN development team is charged with advancing the specification based on input from adopters. This feedback is gathered via the [mailing list](/Contact), which is open to everyone.
 
 The set of standard DSDL definitions is one of the cornerstone concepts of the specification.
-Within the same major version, it can be extended only in the following ways:
+Within the same major version, the specification can be extended only in the following ways:
 
 * A new data type can be added, possibly with default data type ID, as long as the default data type ID doesn't
```

Comment:
@pavel-kirienko I don't know what a data type, default data type or default data type ID are at this point. As a result I can't make much of this. A link would be good.